content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -152,13 +152,13 @@ You are now ready to try the Data Proxy. Go ahead and start your app!
 
 ### Limitations
 
-The Prisma Data Proxy does not support the preview features [Interactive Transactions](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) or [Metrics](/concepts/components/prisma-client/metrics). To use the Prisma Data Proxy, you must fist remove `interactiveTransactions` and `metrics` from the `previewFeatures` section of your `prisma.schema` file.
+The Prisma Data Proxy does not support the preview features [Interactive Transactions](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) or [Metrics](/concepts/components/prisma-client/metrics) yet. To use the Prisma Data Proxy, you must fist remove `interactiveTransactions` and `metrics` from the `previewFeatures` section of your `prisma.schema` file. Running `prisma generate --data-proxy` will error until these preview features are not enabled.
 
 ### Providing the connection string to Prisma Migrate
 
-Currently, the Data Proxy can only be used to query your database with [Prisma Client](/concepts/components/prisma-client).
+Currently, the Data Proxy can only be used to _query_ your database with [Prisma Client](/concepts/components/prisma-client).
```

Comment:
Highlighting the word makes it clearer that this is about the query/migrate, and not about Prisma Client vs. some other Client (which is what I first understood when reading the sentence in isolation).